plugin/pkg/scheduler/factory/plugins.go

```diff
@@ -213,6 +218,14 @@ func RegisterAlgorithmProvider(name string, predicateKeys, priorityKeys util.Str
 		FitPredicateKeys:     predicateKeys,
 		PriorityFunctionKeys: priorityKeys,
 	}
+	temp_name := ""
+	if providerCounter == 1 {
+		temp_name = " | " + name + " | "
+	} else {
+		temp_name = name + " | "
+	}
+	providerCounter += 1
+	AlgorithmProviders += temp_name
```

Comment:
Would be cleaner to use http://golang.org/pkg/strings/#Join over a list of AlgorithmProvider names.